metadata/v1.5.1/step_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Proposal to upgrade mainnet framework to v1.5.1",
+  "description": "This includes changes outlined in https://github.com/aptos-labs/aptos-core/releases/aptos-node-v1.5.1. New public functions in aptos_token to make creating tokens and collections easier.",
```

Comment:
https://github.com/aptos-labs/aptos-core/releases/aptos-node-v1.5.1 is a binary release link, shall we create a framework release tag for this instead?

Reply:
Updated. Not sure why the original link I copied says aptos-node but I went back to the release page and https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.5.1 works as well